llvm/test/CodeGen/AArch64/itofp.ll

```diff
@@ -740,395 +741,571 @@ entry:
-; CHECK-NEXT:    stp d9, d8, [sp, #-64]! // 16-byte Folded Spill
-; CHECK-NEXT:    str x30, [sp, #16] // 8-byte Folded Spill
-; CHECK-NEXT:    stp x22, x21, [sp, #32] // 16-byte Folded Spill
-; CHECK-NEXT:    stp x20, x19, [sp, #48] // 16-byte Folded Spill
-; CHECK-NEXT:    .cfi_def_cfa_offset 64
-; CHECK-NEXT:    .cfi_offset w19, -8
-; CHECK-NEXT:    .cfi_offset w20, -16
-; CHECK-NEXT:    .cfi_offset w21, -24
-; CHECK-NEXT:    .cfi_offset w22, -32
-; CHECK-NEXT:    .cfi_offset w30, -48
-; CHECK-NEXT:    .cfi_offset b8, -56
-; CHECK-NEXT:    .cfi_offset b9, -64
-; CHECK-NEXT:    mov x19, x5
-; CHECK-NEXT:    mov x20, x4
-; CHECK-NEXT:    mov x21, x3
-; CHECK-NEXT:    mov x22, x2
-; CHECK-NEXT:    bl __floattidf
-; CHECK-NEXT:    mov x0, x22
-; CHECK-NEXT:    mov x1, x21
-; CHECK-NEXT:    fmov d8, d0
-; CHECK-NEXT:    bl __floattidf
-; CHECK-NEXT:    mov x0, x20
-; CHECK-NEXT:    mov x1, x19
-; CHECK-NEXT:    fmov d9, d0
-; CHECK-NEXT:    bl __floattidf
-; CHECK-NEXT:    ldp x20, x19, [sp, #48] // 16-byte Folded Reload
-; CHECK-NEXT:    fmov d2, d0
-; CHECK-NEXT:    ldp x22, x21, [sp, #32] // 16-byte Folded Reload
-; CHECK-NEXT:    fmov d0, d8
-; CHECK-NEXT:    ldr x30, [sp, #16] // 8-byte Folded Reload
-; CHECK-NEXT:    fmov d1, d9
-; CHECK-NEXT:    ldp d9, d8, [sp], #64 // 16-byte Folded Reload
-; CHECK-NEXT:    ret
-entry:
-  %c = sitofp <3 x i128> %a to <3 x double>
-  ret <3 x double> %c
-}
-
-define <3 x double> @utofp_v3i128_v3f64(<3 x i128> %a) {
-; CHECK-LABEL: utofp_v3i128_v3f64:
-; CHECK:       // %bb.0: // %entry
-; CHECK-NEXT:    stp d9, d8, [sp, #-64]! // 16-byte Folded Spill
-; CHECK-NEXT:    str x30, [sp, #16] // 8-byte Folded Spill
-; CHECK-NEXT:    stp x22, x21, [sp, #32] // 16-byte Folded Spill
-; CHECK-NEXT:    stp x20, x19, [sp, #48] // 16-byte Folded Spill
-; CHECK-NEXT:    .cfi_def_cfa_offset 64
-; CHECK-NEXT:    .cfi_offset w19, -8
-; CHECK-NEXT:    .cfi_offset w20, -16
-; CHECK-NEXT:    .cfi_offset w21, -24
-; CHECK-NEXT:    .cfi_offset w22, -32
-; CHECK-NEXT:    .cfi_offset w30, -48
-; CHECK-NEXT:    .cfi_offset b8, -56
-; CHECK-NEXT:    .cfi_offset b9, -64
-; CHECK-NEXT:    mov x19, x5
-; CHECK-NEXT:    mov x20, x4
-; CHECK-NEXT:    mov x21, x3
-; CHECK-NEXT:    mov x22, x2
-; CHECK-NEXT:    bl __floatuntidf
-; CHECK-NEXT:    mov x0, x22
-; CHECK-NEXT:    mov x1, x21
-; CHECK-NEXT:    fmov d8, d0
-; CHECK-NEXT:    bl __floatuntidf
-; CHECK-NEXT:    mov x0, x20
-; CHECK-NEXT:    mov x1, x19
-; CHECK-NEXT:    fmov d9, d0
-; CHECK-NEXT:    bl __floatuntidf
-; CHECK-NEXT:    ldp x20, x19, [sp, #48] // 16-byte Folded Reload
-; CHECK-NEXT:    fmov d2, d0
-; CHECK-NEXT:    ldp x22, x21, [sp, #32] // 16-byte Folded Reload
-; CHECK-NEXT:    fmov d0, d8
-; CHECK-NEXT:    ldr x30, [sp, #16] // 8-byte Folded Reload
-; CHECK-NEXT:    fmov d1, d9
-; CHECK-NEXT:    ldp d9, d8, [sp], #64 // 16-byte Folded Reload
-; CHECK-NEXT:    ret
-entry:
-  %c = uitofp <3 x i128> %a to <3 x double>
-  ret <3 x double> %c
-}
-
```

Comment:
Why were these two tests deleted?

Reply:
Hmm... That was a mistake when resolving rebase conflicts... Will add them back.